docs/mechanisms/build.md

```diff
@@ -33,6 +33,18 @@ Moonshot.config do |c|
 ...
 ```
 
+Also **skip_ci_status** is an optional flag would allow us to skip checks 
```

Comment:
This can be '**skip_ci_status** is an optional flag. It would allow us to skip checks'